From 60ca0349eba558b67e90d9f504630309be48e3ff Mon Sep 17 00:00:00 2001
From: Daniel Emminizer <daniel.m.emminizer.civ@us.navy.mil>
Date: Tue, 25 Mar 2025 14:59:36 +0000
Subject: [PATCH 17/29] SDK: Install of GDAL SO optional

**Description:** Attempt to avoid reported WSL error with GDAL install, with INSTALL_THIRDPARTY_LIBRARIES enabled. GDAL shared object install now optional to avoid error if it doesn't exist where expected.

**Testing Performed:** Unable to reproduce issue, but based on reported error this should address the problem.
---
 CMakeUtilities/InstallData.3rd.GDAL.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeUtilities/InstallData.3rd.GDAL.cmake b/CMakeUtilities/InstallData.3rd.GDAL.cmake
index 689fa57d..a3bc9d44 100644
--- a/CMakeUtilities/InstallData.3rd.GDAL.cmake
+++ b/CMakeUtilities/InstallData.3rd.GDAL.cmake
@@ -14,7 +14,7 @@ if(WIN32)
     vsi_install_target(GDAL::GDAL ThirdPartyLibs)
 else()
     get_symlinks("${_GDAL_LIB_DIR}/libgdal.so" _GDAL_LIBS)
-    install(PROGRAMS ${_GDAL_LIBS} DESTINATION "${INSTALLSETTINGS_SHARED_LIBRARY_DIR}" COMPONENT ThirdPartyLibs)
+    install(PROGRAMS ${_GDAL_LIBS} DESTINATION "${INSTALLSETTINGS_SHARED_LIBRARY_DIR}" OPTIONAL COMPONENT ThirdPartyLibs)
 endif()
 
 # Install GDAL plugins
-- 
2.47.1.windows.2

